pmda/rmsf.py

```diff
@@ -222,3 +220,8 @@ def _reduce(res, result_single_frame):
             # update time step in res
             res[2] = k
         return res
+
+    @staticmethod
+    def _negative_rmsf(rmsf):
+        if not (rmsf >= 0).all():
+            raise ValueError("Some RMSF values negative.")
```

Comment:
Keep the longer message. If we have an idea why, tell the user. It also means that we are likely to get less confused/angry mails on the mailing list.